city_scrapers/spiders/chi_transit.py

```diff
@@ -13,10 +13,12 @@
 
 class ChiTransitSpider(Spider):
     name = 'chi_transit'
-    long_name = 'Chicago Transit Authority'
+    agency_id = 'Chicago Transit Authority'
     allowed_domains = ['www.transitchicago.com']
     base_url = 'http://www.transitchicago.com'
-    start_urls = ['https://www.transitchicago.com/board/notices-agendas-minutes/']
+    timezone = 'America/Chicago'
+    START_URL = ['https://www.transitchicago.com/board/notices-agendas-minutes/']
```

Comment:
Is there a reason you have changed the start_url to a nested list? This could be what is throwing Travis off.

Reply:
I'm sure I made lots of mistakes!  I was attempting to steal code from update #378 , and I was following along with the videos on the website.  I'm on slack if you have the time to walk me through what needs to be fixed.